<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientPreferencesView</class>
 <widget class="QDialog" name="ClientPreferencesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(168, 36, 36);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/cuacs logo (b&amp;w).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;DejaVu Sans Mono&quot;;</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>328</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Preferred animal type:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Preferred animal breed:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Preferred age range:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Preferred size:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Ease of training:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Desired animal maintanence fees:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Primarily indoor or outdoor animal:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Do you desire a kid/family friendly animal:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Wilingness to adopt an animal with special needs:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>149</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="typeDropdown">
      <item>
       <property name="text">
        <string>Dog</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bird</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="breedDropdown">
      <item>
       <property name="text">
        <string>Pitbull</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chihuahua</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>German Shepherd</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Poodle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pug</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Husky</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Great Dane</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Greyhound</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ageDropdown">
      <item>
       <property name="text">
        <string>0.6-2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3-6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7-14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15+</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="sizeDropdown">
      <item>
       <property name="text">
        <string>Small</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Large</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="easeDropdown">
      <item>
       <property name="text">
        <string>Low</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>High</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="vetFeesDropdown">
      <item>
       <property name="text">
        <string>$0-$1000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$1000-$2000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$2000-$3000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$4000+</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="envTypeDropdown">
      <item>
       <property name="text">
        <string>Indoor</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Outdoor</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="kidFriendlyDropdown">
      <item>
       <property name="text">
        <string>Yes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>No</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="specialNeedsDropdown">
      <item>
       <property name="text">
        <string>Low</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>High</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
